modules/tournament/src/main/arena/PairingSystem.scala

```diff
@@ -105,4 +105,15 @@ private[tournament] object PairingSystem extends AbstractPairingSystem {
       300 + 1700 * (maxRank - rank) / maxRank
     }
   }
+
+  /* By increasing the factor for high scoring players,
+     * we increase pairing quality for them.
+     * This should increase leader vs leader pairing chances.
+     */
+  private[arena] def scoreFactorFor(players: RankedPlayers): (RankedPlayer, RankedPlayer) => Int = {
```

Comment:
Oh wait, now I see that the `max` is over all the players, not just the pair of players to compare!  I will correct...